src/bluesky/callbacks/fitting.py

```diff
@@ -290,7 +290,7 @@ def _calc_stats(x, y, fields, edge_count=None):
 
         fields["min"] = (x[argmin_y], y_orig[argmin_y])
         fields["max"] = (x[argmax_y], y_orig[argmax_y])
-        (fields["com"],) = np.interp(center_of_mass(y), np.arange(len(x)), x)
+        (fields["com"],) = np.interp(center_of_mass(y - np.min(y)), np.arange(len(x)), x)
```

Comment:
This change decreases the generality.  It imposes an assumption that `y` data is positive.  

PeakStats cannot assume the units of `y` (which includes the sign).  PeakStats must be usable with scaler counts, photocurrents, and any other `float` or `int` value.

Reply:
The challenge to performing any background removal is to identify which background model is to be applied.  This change assumes the model of a positive peak with an additive, constant offset (most common case, encountered when using a scaler).

Other models might consider positive peak against a sloped background, negative peak versions of these two, or even `y` data that has comparably strong positive and negative values.

For PeakStats to be general, it cannot perform background removal assuming only one of these models, even if that model is common.